config/workbox.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  globDirectory: 'build/',
+  globPatterns: ['**/*.{css,js,html}'],
+  swDest: 'build/service-worker.js',
+  swSrc: 'build/services/service-worker/home.js',
+  maximumFileSizeToCacheInBytes: 10 * 1024 * 1024
```

Comment:
can you add a comment here or add a description in the commit to explain why we need to set this key?